gsy_e_sdk/setups/asset_api_scripts/rest_basic_strategies.py

```diff
@@ -153,16 +153,20 @@ def post_bid_offer(self, rate_index=0):
                 )
 
             # Storage assets
-            buy_energy = asset_info.get("energy_to_buy")
-            if buy_energy:
+            buy_energy = asset_info.get("energy_to_buy", 0)
+            bid_energy = asset_info.get("energy_active_in_bids", 0)
+            if buy_energy > 0 or bid_energy > 0:
                 buy_rate = self.asset_strategy[area_uuid]["buy_rates"][rate_index]
-                self.add_to_batch_commands.bid_energy_rate(
+                buy_energy = buy_energy + bid_energy
```

Comment:
So you are offering the energy, that you placed bids for already?

Reply:
Thanks for your review!
To answer your question:
Yes, I'm accounting for the `energy_to_buy` **and** the `energy_active_in_bids` as the total energy requirements.

Then, a new bid will be made considering the whole amount of energy that is required, replacing the old bid.
This allows posting a new (updated) bid that overrides the old one but with an updated rate.

I don't understand why this would be dangerous. Maybe the name I gave the `bid_energy`variable is misleading, as it refers to the `energy_active_in_bids` value received in the area_dict on the corresponding tick. Maybe I got you wrong. Could you please elaborate on that case?

Reply:
Yes, I got all that and I see my confusion now. Have a look one line below. You want to buy energy here, but call `self.add_to_batch_commands.offer_energy_rate()` . I think it should be `self.add_to_batch_commands.bid_energy_rate()`. 
Thanks!

Reply:
Agreed with Hannes' comment and also want to suggest to rename the variable `bid_energy` to `open_bids_energy` in order for the intent to be immediately clear. Thanks!

Reply:
@hannesdiedrich You're right! Thanks for pointing that out.

I addressed both observations on the latest commit.